### PR TITLE
Feat/fetch upstream

### PR DESCRIPTION
**Pull request by jbust23**

### CODE REVIEW

**Review by Mejiro-McQueen — APPROVED** [review]
I can't say much about the OpenMCT changes.
Shouldn't affect us if Joe is working off of the upstream.

File: doc/source/conf.py [hunk 1]
```diff
@@ -65,9 +65,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'2.3.5'
+version = u'2.3.6-rc1'
```
Comment (cSchubes) [inline doc/source/conf.py:68]: should we change to our version?

File: doc/source/conf.py [hunk 2]
```diff
@@ -65,9 +65,9 @@
...
 # The full version, including alpha/beta/rc tags.
-release = u'2.3.5'
+release = u'2.3.6-rc1'
```
Comment (cSchubes) [inline doc/source/conf.py:70]: should we change to our version?

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,116 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Issue #358 - OpenMCT Plugin support for multiple clients and filtered telemetry [`#426`](https://github.com/NASA-AMMOS/AIT-Core/pull/426)
+- Issue #401 - Minor fixes to issues found in master [`#422`](https://github.com/NASA-AMMOS/AIT-Core/pull/422)
+- Issue #371 - Fix calls to logger in util pkl handling [`#423`](https://github.com/NASA-AMMOS/AIT-Core/pull/423)
+- Issue #420 - Add Packet Accumulator [`#418`](https://github.com/NASA-AMMOS/AIT-Core/pull/418)
+- Issue #419 - Add Packet Padder Plugin [`#417`](https://github.com/NASA-AMMOS/AIT-Core/pull/417)
+- Issue #328 - Update getDefaultDict to return CmdDict extensions when applicable [`#404`](https://github.com/NASA-AMMOS/AIT-Core/pull/404)
+- Issue #304 - Improve time handling in database insert method [`#421`](https://github.com/NASA-AMMOS/AIT-Core/pull/421)
+- Issue #371 - Testing nested includes during test for 'dirty' (pickle) [`#416`](https://github.com/NASA-AMMOS/AIT-Core/pull/416)
+- Issue #394 - Add DSN paths to ait-create-dirs [`#396`](https://github.com/NASA-AMMOS/AIT-Core/pull/396)
+- issue #401 - Fix ArrayType encode bug [`#402`](https://github.com/NASA-AMMOS/AIT-Core/pull/402)
```
Comment (cSchubes) [inline CHANGELOG.md:25]: it's hilarious they include this given they already rolled it back, lmao

**Review by cSchubes — APPROVED** [review]
approving but see version comments

File: doc/source/conf.py (hunk 1 above)
Comment (jbust23) [reply doc/source/conf.py:68]: I didnt update it here because its just for the docs but I can eventually update this if needed

**Pull request merged**